config.json

```diff
@@ -44,7 +44,8 @@
   ],
   "ignored": [
     "_test",
-    "docs"
+    "docs",
+    "img"
```

Comment:
I thought about it a bit. I'm not hugely concerned. In fact it might be less suprising if we keep it. Imagine if we removed this change, then later down the road I take a look and wonder "well the img dir is here, but why is it not being excluded...?"